persistent/tests/test_persistence.py

```diff
@@ -586,6 +666,9 @@ def test___getattribute___p__names(self):
         for name in NAMES:
             getattr(inst, name)
         self._checkMRU(jar, [])
+        # _p_mtime is special, it activates the object
+        getattr(inst, '_p_mtime')
+        self._checkMRU(jar, [OID])
```

Comment:
It doesn't seem like a bug in the C implementation. The `Per_get_mtime` function that backs `_p_mtime` is very specific about unghostifying the object and moving it to the front of the MRU list. 

I did run into code in the wild that uses `_p_mtime` as a convenient way to tell if an object has actually ever been saved, whether or not it is now changed. Checking the `_p_oid` is not sufficient: An object can have an `_p_oid` if `Connection.add` has been called even before the transaction is commited, but `_p_mtime` only becomes available after the transaction is committed.